R/reportMOFEX.R

```diff
@@ -69,8 +69,12 @@ reportMOFEX <- function(gdx,gdx_ref=NULL,file=NULL,scenario='default') {
   fuel2pe.m <- fuel2pe.m[,t,]
   budget.m <- budget.m[,t,]
   tradebal.m <- tradebal.m[,t,]
-  p_cint          <- collapseNames(mselect(p_cint,all_enty="co2"))
-  getSets(p_cint) <- gsub("all_enty1","all_enty",getSets(p_cint))
+  if(getSets(p_cint)[[3]] == "emiTe"){
+      p_cint          <- collapseNames(mselect(p_cint,emiTe="co2"))
+  } else {
+      p_cint          <- collapseNames(mselect(p_cint,all_enty="co2"))
+      getSets(p_cint) <- gsub("all_enty1","all_enty",getSets(p_cint))
```

Comment:
Is it correct that this is only in the `else` branch?

Reply:
Yes see comment above